pkg/graph/schema.graphql

```diff
@@ -286,6 +286,18 @@ type CedarSystemDetails {
  urls: [CedarURL!]!
 }
 
+type SystemWorkspace {
+  linkedRequests: CedarLinkedRequests!
+}
```

Comment:
I would think it would be safe to just place `linkedRequests` as a type directly on `type CedarSystem`, rather than making a separate resolver for it (since the system workspace already will depend on `type CedarSystem` for things like `isBookmarked`, the system name, description, acronym, etc.

Reply:
this has been refactored!